rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -160,7 +162,7 @@ def test_reconciliation_query(self):
     self._insert_order(p_on_time, 'GoodOrder', BIOBANK_TESTS[:4], order_time,
                        finalized_tests=BIOBANK_TESTS[:3], kit_id='kit1', tracking_number='t1',
                        collected_note=u'\u2013foo', processed_note='bar', finalized_note='baz')
-    self._insert_samples(p_on_time, BIOBANK_TESTS[:2], ['GoodSample1', 'GoodSample2'],
+    self._insert_samples(p_on_time, BIOBANK_TESTS[:2], ['GoodSample1', 'GoodSample2'], 'KIT',
```

Comment:
We should use different values for the order identifier for different samples... for now it doesn't make much difference, but when it comes time to do the reconciliation report it will.

Reply:
addressed below.

Reply:
done